docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -250,65 +227,419 @@ access_entries = {
+
+export DB_HOST="$(terraform output -raw postgres_endpoint)"
+export DB_ROLE_ARN="$(terraform output -raw aurora_role_arn)"
+
+# OpenSearch
+export OPENSEARCH_HOST="$(terraform output -raw opensearch_endpoint)"
+export OPENSEARCH_ROLE_ARN="$(terraform output -raw opensearch_role_arn)"
+export CAMUNDA_ZEEBE_SERVICE_ACCOUNT_NAME="$(terraform console <<<local.camunda_zeebe_service_account | jq -r)"
+export CAMUNDA_OPERATE_SERVICE_ACCOUNT_NAME="$(terraform console <<<local.camunda_operate_service_account | jq -r)"
+export CAMUNDA_TASKLIST_SERVICE_ACCOUNT_NAME="$(terraform console <<<local.camunda_tasklist_service_account | jq -r)"
+export CAMUNDA_OPTIMIZE_SERVICE_ACCOUNT_NAME="$(terraform console <<<local.camunda_optimize_service_account | jq -r)"
+```
+
+:::note IRSA users
+
+To authenticate and authorize access to PostgreSQL and OpenSearch, **you do not need to export the PostgreSQL or OpenSearch passwords**, IRSA will handle the authentication.
+
+**However**, you will still need to export the relevant usernames and other settings to Helm.
+
+:::
+
+  </TabItem>
+</Tabs>
+
+Ensure that you use the actual values you passed to the Terraform module during the setup of PostgreSQL and OpenSearch.
+
+### Create the database and associated access
+
+As you now have a database, you need to create dedicated databases for each Camunda component and an associated user that have a configured access. Follow these steps to create the database users and configure access.
+
+You can access the created database in two ways:
+
+1. **Bastion host**: Set up a bastion host within the same network to securely access the database.
+2. **Pod within the EKS cluster**: Alternatively, deploy a pod in your EKS cluster equipped with the necessary tools to connect to the database.
+
+The choice depends on your infrastructure setup and security preferences. In this tutorial, we'll use a pod within the EKS cluster to configure the database.
+
+1. **Set the environment variables**: In your terminal, set the necessary environment variables that will be substituted in the setup manifest.
+
+```bash
+export AURORA_ENDPOINT=$(terraform output -raw postgres_endpoint)
+export AURORA_PORT=5432
+export AURORA_DB_NAME="$DEFAULT_DB_NAME"
+
+# PostgreSQL Credentials (replace with your own values from the #postgresql-module-setup step)
+export AURORA_USERNAME="$(terraform console <<<local.aurora_master_username | jq -r)"
+export AURORA_PASSWORD="$(terraform console <<<local.aurora_master_password | jq -r)"
+```
+
+A **Kubernetes job** will connects to the database and creates the necessary users with the required privileges. The script installs the necessary dependencies and runs SQL commands to create the IRSA user and assign it the correct roles and privileges.
+
+2. **Create a copy of the manifest**: Save the above manifest to a file, for example, `setup-postgres-create-db.yml`.
```

Comment:
in this section, wouldn't you first create the secret and then the job that depends on those secrets?

Reply:
ah just saw, you first tell to create and only later to apply.